client/src/components/NavBar.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import HomeIcon from "../assets/icons/home-inactive.svg";
+import SearchIcon from "../assets/icons/search-inactive.svg";
+import AddIcon from "../assets/icons/plus-inactive.svg";
+import HeartIcon from "../assets/icons/heart-inactive.svg";
+
+import styled from "styled-components/macro";
+
+const Footer = styled.footer`
+  height: 62px;
+  background: var(--primary-color);
+  display: flex;
+  justify-content: space-around;
+`;
+
```

Comment:
even if its in the footer, isnt it mainly a navigation of the site?
```suggestion
const Footer = styled.nav`
  height: 62px;
  background: var(--primary-color);
  display: flex;
  justify-content: space-around;
`;

```